draft-ietf-quic-transport.md

```diff
@@ -1002,7 +1002,9 @@ When an endpoint issues a connection ID, it MUST accept packets that carry this
 connection ID for the duration of the connection or until its peer invalidates
 the connection ID via a RETIRE_CONNECTION_ID frame
 ({{frame-retire-connection-id}}).  Connection IDs that are issued and not
-retired are considered active; any active connection ID can be used.
+retired are considered active; any active connection ID is valid for use at any
+time, including during the handshake.  This includes the connection ID issued by
```

Comment:
Maybe add "in any packet type", considering that the original issue was if the CID of a Handshake packet can change?

Note: CID of a server-sent Initial packet might also use a CID delivered by a NCID frame, when 0-RTT is being used.

Reply:
Discard-Initial currently says:
"A client stops both sending and processing Initial packets when it sends its first Handshake packet."

If a client processed an Initial, Handshake and 0.5RTT from the server, then it could start using the CID in NEW_CONNECTION_ID in Initial packet if it sent the Initial packet right before the Handshake packet.

Of course, if the Initial was dropped it wouldn't matter, so this is a bit irrelevant, but I wanted to make sure others realized it was not just Handshake, because I missed that previously.

Reply:
Both those Initial cases are interesting, but probably not worth extra text :)

The "any packet type" comment is worth the text.

Reply:
Good call, "during the handshake" is unclear in referring to a time or a packet type. Updated!

Reply:
Settled on just noting "any packet type" since the presence of Initial and Handshake packets implies that you're using these during the handshake.